{"name": "test_locked_out_user", "status": "passed", "description": "Test login with a locked-out user (expected failure)", "steps": [{"name": "Logging in with locked_out_user", "status": "passed", "start": 1738274252046, "stop": 1738274255418}, {"name": "Verify login fails with the expected error message", "status": "passed", "start": 1738274255418, "stop": 1738274255447}], "start": 1738274252046, "stop": 1738274255448, "uuid": "ae1fb680-f3b3-4691-80d0-48eff80f5fa1", "historyId": "ed79c1a0b2bfef01e0e3c3da630db092", "testCaseId": "ed79c1a0b2bfef01e0e3c3da630db092", "fullName": "tests.test_login#test_locked_out_user", "labels": [{"name": "story", "value": "Locked Out User"}, {"name": "feature", "value": "Login"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "Ram"}, {"name": "thread", "value": "27844-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}